spec/classroom_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'all_methods'
+
+describe Classroom do
+  context 'when creating a new classroom' do
+    classroom = Classroom.new('Math')
+
+    it 'should have a label' do
+      expect(classroom.label).to eql('Math')
+    end
+
+    it 'should have no students' do
+      expect(classroom.students).to eql([])
+    end
+  end
```

Comment:
- [ ] You did well testing your initializer for a Classroom class, But as part of the main requirements for this milestone, we need to have an 'it' block for each class method (at least). Please add a test case for your `add_student` (`def add_student(student))`) method.

Reply:
- Well resolved 👍🏽